packages/editor-ui/src/components/InviteUsersModal.vue

```diff
@@ -333,7 +333,7 @@ export default defineComponent({
 			}
 		},
 		goToUpgradeAdvancedPermissions() {
-			void this.uiStore.goToUpgrade('settings-users', 'upgrade-advanced-permissions');
+			void this.uiStore.goToUpgrade('advanced-permissions', 'upgrade-advanced-permissions');
```

Comment:
this looks like it fixes an unrelated bug. is there a ticket for that?

Reply:
It's related to this. Should this be a separate PR?

![Capture 2023-12-19 at 16 25 35@2x](https://github.com/n8n-io/n8n/assets/44588767/bd257416-01ca-4a38-b3b8-9a1bd4485881)